src/activity_handlers.cpp

```diff
@@ -292,6 +293,8 @@ void activity_handlers::butcher_finish( player_activity *act, player *p )
             g->m.spawn_item( p->pos(), "bone", bones, 0, age );
             p->add_msg_if_player( m_good, _( "You harvest some usable bones!" ) );
         }
+    } else if( meat == "bone_tainted" ) {
```

Comment:
This check seems useless here. The player will harvest tainted bones no matter what if `bones > 0` (see the existing code above). If they harvest tainted bones on success (with the appropriate message), why suppress the message on failure?